README.md

```diff
@@ -14,8 +14,13 @@ composer require lmc/aerospike-cache
 ```
 
 ## Usage
+```php
+$aerospike = new \Aerospike(['hosts'=>[['addr'=>'127.0.0.1', 'port'=>3000]]]);
```

Comment:
Even though it is not the concern of this adapter, I would mention, that creating a new `Aerospike` instance will immediately connects the aerospike server, which might take a while and it should be in `lazy` abstraction.

Reply:
Added to Readme